mail.php

```diff
@@ -0,0 +1,64 @@
+<?php
+if(isset($_POST['submit'])){
```

Comment:
Format this document perfectly. Make sure you provide required spacing.
for eg. **if (isset($_POST['submit'])) {**